### PR TITLE
Meeting Minutes for 2024-10-30

### PR DESCRIPTION
**Pull request by santosomar**
This pull request includes the meeting minutes for the CSAF Technical Committee Working Meeting held on October 30, 2024. The minutes document the participants, agenda, discussions, and motions passed during the meeting.

Key changes include:

* Addition of the meeting agenda and participants list, including roles and affiliations.
* Documentation of discussions on false positives, product description relationships, and the use of MK Docs for the documentation website.
* Recording of motions passed to approve various pull requests and include support for SSVC and EPSS in CSAF 2.1.
* Note on the next meeting date and the regular schedule for monthly meetings.

### CODE REVIEW

**Review by sthagen — APPROVED** [review]
LGTM.

File: meeting_minutes/2024/2024-10-30.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role          |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|
+| Christoph  | Plutte      | Ericsson                                                    | Voting Member |
+| Denny      | Page        | Individual                                                  | Voting Member |
+| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member |
+| Feng       | Cao         | Oracle                                                      | Voting Member |
+| JD         | Stefaniak   | Dell                                                        | Member        |
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:24]: I didn't record Dina, Feng and JD, when I calculated quorum. Did they join later?

File: meeting_minutes/2024/2024-10-30.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Voting Member |
+| Michael    | Reeder      | Dell                                                        | Voting Member |
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:26]: I didn't record Michael, when I calculated quorum. Did he join later?
However, we are missing Martin who definitely was there.
```suggestion
| Martin      | Prpic          | Red Hat                                                | Voting Member |
| Michael    | Reeder      | Dell                                                        | Voting Member |

File: meeting_minutes/2024/2024-10-30.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| Omar       | Santos      | Cisco                                                       | Chair         |
+| Rhonda     | Levy        | Cisco                                                       | Voting Member |
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:28]: I didn't record Rhonda, when I calculated quorum. Did she join later?

File: meeting_minutes/2024/2024-10-30.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- Roll call
+- Secretary nomination
+- Review GitHub Issues for TC Discussion:  https://github.com/oasis-tcs/csaf/issues
+- Discuss next steps.
+- Adjourn
+
+**Quorum was reached.**
+
+## Meeting Notes
+- Christophe led a discussion about false positives and best practices for describing them in CSAF.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:52]:
```suggestion
- Christoph led a discussion about false positives and best practices for describing them in CSAF.

File: meeting_minutes/2024/2024-10-30.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- There was a discussion about using relationships and nested relationships in product descriptions.
+- The team is considering using MK Docs or a permutation of it for the documentation website.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:54]:
```suggestion
- The team is considering using MK Docs or a derivate / similar product of it for the documentation website.

File: meeting_minutes/2024/2024-10-30.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+- The timeline plan includes a feature close, November editor revision, merging of open motions, and - preparation for the next meeting.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:55]: Even though it is now clear that we won't be able to make it, I think the plan was to proceed to an CSD01.
```suggestion
- The timeline plan includes a feature close, November editor revision, merging of open motions, and - preparation of the Committee Specification Draft 01 for the next meeting.

File: meeting_minutes/2024/2024-10-30.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+- The December meeting may be moved to December 18th to accommodate the holidays.
+- Thomas sets the motion to approved pull request #814 https://github.com/oasis-tcs/csaf/pull/814. Thomas presented a motion to incorporate the changes outlined in this PR. The motion was seconded by Sonny and, following a unanimous lack of discussion or objections, it was carried.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:57]: I guess it was not approve at the time, right?
```suggestion
- Thomas sets the motion to approve pull request #814 https://github.com/oasis-tcs/csaf/pull/814. Thomas presented a motion to incorporate the changes outlined in this PR. The motion was seconded by Sonny and, following a unanimous lack of discussion or objections, it was carried.

File: meeting_minutes/2024/2024-10-30.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+- [Pull Request #810](https://github.com/oasis-tcs/csaf/pull/810). A motion was moved by Omar to include the changes suggested in this pull request. The motion was seconded by Denny. The motion passed.
+- [Pull Request #805](https://github.com/oasis-tcs/csaf/pull/805).  A motion was moved by Omar to include the changes suggested in this pull request. The motion was seconded by Thomas Schaffer. The motion passed.
+- Motion passed to include support for SSVC in CSAF 2.1 as an optional field, as discussed in [Issue #803](https://github.com/oasis-tcs/csaf/issues/803). A motion was moved by Omar to include the changes suggested in this pull request. The motion was seconded by Michael. The motion passed.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:60]: Sorry - PR is still missing.
```suggestion
- Motion passed to include support for SSVC in CSAF 2.1 as an optional field, as discussed in [Issue #803](https://github.com/oasis-tcs/csaf/issues/803). A motion was moved by Omar to include the changes suggested in this issue. The motion was seconded by Michael. The motion passed.

File: meeting_minutes/2024/2024-10-30.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+- Motion passed to include support for EPSS in CSAF 2.1 as an optional field, as described in [Issue #622](https://github.com/oasis-tcs/csaf/issues/622). A motion was moved by Thomas to include the changes suggested in this pull request, during the CSAF TC monthly meeting on 2024-10-30. The motion was seconded by Omar. The motion passed.
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:61]: Same here: Sorry - no PR yet.
```suggestion
- Motion passed to include support for EPSS in CSAF 2.1 as an optional field, as described in [Issue #622](https://github.com/oasis-tcs/csaf/issues/622). A motion was moved by Thomas to include the changes suggested in this issue, during the CSAF TC monthly meeting on 2024-10-30. The motion was seconded by Omar. The motion passed.

**Review by tschmidtb51 — COMMENTED** [review]
Some comments. I'm not too sure about my comments regarding the participants (as I stopped recording the name as soon as we had quorum :face_with_peeking_eye:). Maybe, someone can help here.

File: meeting_minutes/2024/2024-10-30.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+Inability to register attendees due to OASIS system challenges.
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role          |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|
+| Christoph  | Plutte      | Ericsson                                                    | Voting Member |
+| Denny      | Page        | Individual                                                  | Voting Member |
+| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member |
+| Feng       | Cao         | Oracle                                                      | Voting Member |
```
Comment (tschmidtb51) [inline meeting_minutes/2024/2024-10-30.md:23]: Feng was not present as stated by himself
```suggestion

File: meeting_minutes/2024/2024-10-30.md (hunk 2 above)
Comment (santosomar) [reply meeting_minutes/2024/2024-10-30.md:25]: He was there:

image

File: meeting_minutes/2024/2024-10-30.md (hunk 3 above)
Comment (santosomar) [reply meeting_minutes/2024/2024-10-30.md:27]: She opened the meeting and then left. I will remove her from the list.

**Comment by santosomar** [comment]
Motion sent and seconded by Stefan: https://groups.oasis-open.org/discussion/motion-to-approve-october-2024-meeting-minutes

**Comment by santosomar** [comment]
Motion passed. https://groups.oasis-open.org/discussion/motion-to-approve-october-2024-meeting-minutes

**Pull request merged**